src/util.ts

```diff
@@ -211,13 +214,31 @@ export class Note extends GObject.Object {
     this.width = note.width;
     this.height = note.height;
     this.open = note.open ?? false;
+
+    // @ts-expect-error incorrect types
+    this.bind_property_full(
+      "content",
+      this,
+      "title",
+      GObject.BindingFlags.SYNC_CREATE,
+      (_, content) => {
+        if (!content) return [false, ""];
+        return [true, content.substring(0, 24)]
```

Comment:
You can do this so that it returns the first line if it's there.

```suggestion
        return [true, content.split("\n")[0].substring(0, 24)]
```

Also this should add an ellipsis (…) if the content has been truncated.